### DISCUSSION TITLE
Footer unclear (Simple theme)

### DISCUSSION THREAD

**Discussion by henkvantijen** [GetPublii/Publii#1700] [Help]
My client wants a common  black footer under all pagesof the site. 
I added to  T&P > Custom CSS
  .footer{
        background: black;
  }
  .footer p {color: white;}

This works, I get a black area under every page. 
To add content to the footer, I added text to T&P > Custom HTML > Footer , but this text appears _below_ the black footer, so  it's more of a below-footer?

[ I found out the actual content of the footer can be set at Theme > Footer , this for other newbies who might be reading this]

**Comment by bjazmoore** [comment]
So did you answer your own question?  Is it working now?

**Reply by henkvantijen** [reply]
https://github.com/GetPublii/Publii/discussions/2551
